package.json

```diff
@@ -44,5 +44,8 @@
         "last 1 versions",
         "IE 11",
         "not dead"
-    ]
+    ],
+    "scripts": {
+        "start": "lerna run start --parallel"
```

Comment:
> Stream output from child processes immediately, prefixed with the originating package name. This allows output from different packages to be interleaved. [...] but completely disregards concurrency and topological sorting, running a given command or script immediately in all matching packages with prefixed streaming output. This is the preferred flag for long-running processes [...]

You are using this on `run start`, why ?

Reply:
This allows to run `npm start` in the root of react-vapor. It'll run in parallel `npm start` in all the sub packages (react-vapor and vapor) in parallel which is pretty convenient.